### PR TITLE
add workflow to test postman collections against docker compose scripts

### PR DESCRIPTION
**Pull request by jeffreyscarpenter**
Add a GitHub Actions workflow to run Stargate Postman collections using the Postman CLI against backends started using docker compose scripts.

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

**Comment by jeffreyscarpenter** [comment]
This initial version will run against the latest released docker images tagged `v2`. We can modify this if desired to build the current snapshot and generate local docker images to be used by the docker compose scripts.

File: .github/workflows/postman-docker.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          - collection: restapi-users
+            collection-id: 17930693-47ab5f0d-407e-48cf-aa11-c51d129f1eef
+            environment-id: 17930693-2f355f7b-123d-4bca-9a9c-7d82222ca49d
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Postman CLI
+        run: |
+          curl -o- "https://dl-cli.pstmn.io/install/linux64.sh" | sh
+      - name: Login to Postman CLI
+        run: postman login --with-api-key ${{ secrets.POSTMAN_API_KEY }}
```
Comment (ivansenic) [inline .github/workflows/postman-docker.yaml:48]: wha about using a dedicated action: https://github.com/marketplace/actions/newman-cli-postman-action

File: .github/workflows/postman-docker.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      - name: Start Backend
```
Comment (ivansenic) [inline .github/workflows/postman-docker.yaml:49]: separating steps with a blank line helps a lot I was struggling for a while to understand where is the docker compose started :)

**Review by ivansenic — APPROVED** [review]
Let's merge and then test once it's on the main..

File: .github/workflows/postman-docker.yaml (hunk 1 above)
Comment (jeffreyscarpenter) [reply .github/workflows/postman-docker.yaml:48]: idk, this step actually was autogenerated for me by postman.

File: .github/workflows/postman-docker.yaml (hunk 2 above)
Comment (jeffreyscarpenter) [reply .github/workflows/postman-docker.yaml:49]: Agree, will add on the next update.

**Comment by jeffreyscarpenter** [comment]
> Let's merge and then test once it's on the main..

agree. I have been testing it locally with https://github.com/nektos/act but would like to see how it does in GitHub.

**Pull request merged**